one/util.py

```diff
@@ -404,7 +404,7 @@ def run_main(main_func):
 
 def get_datastore_args(expected_arg_count=2):
     arg_image_id = None
-    if one_version_larger(6, 6):
+    if one_version_larger(6, 8):
```

Comment:
why 6.8? this was introduced in 6.8
and the check is if the opennebula version is larger than 6.6, which would be 6.8...

Reply:
I tested the addon on the latest version 6.6.3: if you leave the definition of version >6.6, the addon will not work (the default configuration does not include the "-i" option in the datastore mad, which is responsible for the new format of the datastore driver)

Reply:
Oh, I see now that the problem is.

the `one_version_larger()` function didn't do what I expected it todo, it should ignore patch levels if not specified,
So your `6.6.3` acted like it was some `6.8` version...

should be fixed with: 82b785ccc7cd9e557155c2bc5910093dfd7ef282